src/bindgen/index.md

```diff
@@ -20,6 +20,8 @@ use Rust in a C/C++ codebase in the first place!
 
 ## Getting Set Up
 
+> **Note:** Bindgen **requires** clang. Installation instructions can be found [here](https://github.com/rust-lang-nursery/rust-bindgen/blob/master/book/src/requirements.md#installing-clang-39)
```

Comment:
hmm, the master branch of bindgen renders the same way when using `mdbook serve` locally
Strangely, only on this file.